src/optimize/base_optimizer.js

```diff
@@ -122,12 +118,19 @@ export default class BaseOptimizer {
           allChunks: true
         }),
 
-        new CommonsChunkPlugin({
+        new webpack.optimize.CommonsChunkPlugin({
           name: 'commons',
-          filename: 'commons.bundle.js'
+          filename: 'commons.bundle.js',
+          minChunks: 2,
         }),
 
-        new NoEmitOnErrorsPlugin(),
+        new webpack.optimize.CommonsChunkPlugin({
+          name: 'vendor',
```

Comment:
should probably be `vendors`

Reply:
Thanks, missed it when I changed from `vendor` to `vendors`.